v22.1/alter-backup.md

```diff
@@ -0,0 +1,95 @@
+---
+title: ALTER BACKUP
+summary: Use the ALTER BACKUP statement to add `NEW_KMS` encryption keys to backups.
+toc: true
+docs_area: reference.sql
+---
+
+{% include enterprise-feature.md %}
+
+<span class="version-tag">New in v22.1:</span> The `ALTER BACKUP` statement allows for new KMS encryption keys to be applied to existing encrypted backups. Each `ALTER BACKUP` statement must include the new KMS encryption key with `NEW_KMS`, and use `WITH OLD_KMS` to refer to at least one of the KMS URIs that were originally used to encrypt the backup.
+
+After an `ALTER BACKUP` statement successfully completes, subsequent `BACKUP`, `RESTORE`, and `SHOW BACKUP` statements can use any of the existing or new KMS URIs to decrypt the backup.
+
+CockroachDB supports AWS and Google Cloud KMS keys. For more detail on encrypted backups and restores, see [Take and Restore Encrypted Backups](take-and-restore-encrypted-backups.html).
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/master/grammar_svg/alter_backup.html %}
+</div>
+
+## Parameters
+
+Parameter         | Description
+------------------+-------------------------------------------------------------------------------------------------------------------------
+`subdirectory`    | The subdirectory that contains the target backup at the given `collectionURI`.
+`LATEST`          | The most recent backup at the given `collectionURI`.
+`collectionURI`   | The URI that holds the backup collection.
+`ADD NEW_KMS`     | Apply the new KMS encryption key to the target backup.
+`WITH OLD_KMS`    | Reference one of the existing KMS URI(s) originally used to encrypt the backup.
+`kmsURI`          | The [URI](take-and-restore-encrypted-backups.html#uri-formats) for the KMS key.
+
+## Required privileges
+
+- `ALTER BACKUP` can only be run by members of the [`admin` role](security-reference/authorization.html#admin-role). By default, the `root` user belongs to the `admin` role.
+- `ALTER BACKUP` requires full read and write (including delete and overwrite) permissions to the target cloud storage bucket.
+
+The backup collection's URI does **not** require the [`admin` role](security-reference/authorization.html#admin-role) when using `s3` or `gs` using `SPECIFIED` credentials. The backup collection's URI **does** require the [`admin` role](security-reference/authorization.html#admin-role) when using `s3` or `gs` with `IMPLICIT` credentials.
+
+We recommend using [cloud storage for bulk operations](use-cloud-storage-for-bulk-operations.html).
+
+## Examples
+
+See [Use Cloud Storage for Bulk Operations](use-cloud-storage-for-bulk-operations.html) for more detail on authenticating to your cloud storage bucket.
+
+### Add an AWS KMS key to an encrypted backup
```

Comment:
is it worth clarifying that when running an `ALTER BACKUP <not-LATEST> IN <collection>` or an `ALTER BACKUP <resolved-path-in-collection>` the fully resolved paths must point to a full backup in the collection? In other words the customer must `ALTER` the base backup of the chain, and will not be able to do so on an incremental backup in the chain. @DarrylWong does this seem right?

Reply:
Yeah that seems right